pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial

```diff
@@ -0,0 +1,28 @@
+---
+title: Spread the word
+layout: wide
+nav: promote_nav
+---
+
+{{ signup_button }}
```

Comment:
Oh, is `{{ }}` the new syntax for including partials in a `.partial`?

Reply:
Oh, yep! Sorry I wasn't clear about that; this is a follow-up to https://github.com/code-dot-org/code-dot-org/pull/28118